### ISSUE TITLE
Exporting routes to PDF throwing an error.

### ISSUE THREAD

**Issue by swarchen** [google/cfr#36]
![Dec-14-2022 14-12-26](https://user-images.githubusercontent.com/11629463/207520070-8b5dd87c-dd37-42b9-a79e-f3b15ba35ee3.gif)

```
Generating PDF
Error generating report
Cannot read properties of undefined (reading 'styles')
```

Does anyone encounter the same issue as mine

**Comment by jasonmccollumwoolpert** [comment]
Can you confirm what version of the application you're using? In the recently released 4.4.0 branch we did some improvements to the PDF export process so want to double check if this is still an issue in the latest release or not.

**Comment by swarchen** [comment]
Hi @jasonmccollumwoolpert I'm using 4.4.0 right now. Let me try to use the previous version to check if this is the issue of the latest version.

**Comment by jasonmccollumwoolpert** [comment]
Just released v4.4.1, which should address this issue. Can you try updating your deployment and confirm?

**Comment by swarchen** [comment]
Hi @jasonmccollumwoolpert 
It works perfectly! Thank you for the help
